doc/explanations/get-started.rst

```diff
@@ -0,0 +1,719 @@
+==================================
+Getting started with native tokens
```

Comment:
This documentation should probably live in the cardano-node repository.

Reply:
deleted